src/instruments/src/EFB/Components/Progress/Progress.tsx

```diff
@@ -31,6 +31,7 @@ export const ProgressBar: React.FC<ProgressBarProps> = ({
     completedBarEnd,
     completedBarBegin,
     completedBarBeginValue,
+    rawThrottlePosition,
```

Comment:
This might not only be used for throttles. Let's rename this prop.

Reply:
The naming is fine as its the throttle's position

Reply:
This component is in a folder that is for generic components.

Reply:
I forgot about that, thank you. Fixed with latest commit.